src/proj7BittingCerratoCohenEllmer/bantam/Main.java

```diff
@@ -3,9 +3,10 @@
 import proj7BittingCerratoCohenEllmer.bantam.lexer.Token;
 import proj7BittingCerratoCohenEllmer.bantam.util.CompilationException;
 import proj7BittingCerratoCohenEllmer.bantam.util.ErrorHandler;
-
-public class Main {
-        /**
+import proj7BittingCerratoCohenEllmer.bantam.util.Error;
+//TODO : should this be renamed to "ScannerTest" ?
+public class Main { 
+    /**
```

Comment:
This could also be moved into the Scanner.java file as per bullet 8 of Dales instructions